### PR TITLE
logging and browser helper on selene implemented

### PR DESCRIPTION
**Pull request by yefim-mironyuk**
Browser helper methods tested and work correct.

### CODE REVIEW

File: utils/browser_helper.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
+from loguru import logger
+from selene import be
+from selenium.common.exceptions import NoSuchElementException, ElementNotVisibleException, \
```
Comment (relaxing-observer) [inline utils/browser_helper.py]: Maybe we should use WebDriverException for all try/except blocks?

File: utils/browser_helper.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    ElementNotInteractableException, TimeoutException
+
+
+class ElementInteractions:
+    def __init__(self, browser):
+        self.browser = browser
+
```
Comment (relaxing-observer) [inline utils/browser_helper.py:12]: Should we initialize instance of ElementInteractions class or stay out of it (remove init method and self from every method below?

Comment (relaxing-observer) [inline utils/browser_helper.py:12]: Need to set logging config from logging.py

Comment (yefim-mironyuk) [reply utils/browser_helper.py:12]: ### Closed. 
We decided to create instance factory.

File: utils/browser_helper.py (hunk 1 above)
Comment (yefim-mironyuk) [reply utils/browser_helper.py]: Not working. Selene has its own exception. TimeoutException only.

**Review by kit-91-daniil — APPROVED** [review]

File: utils/logging.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from loguru import logger
+
+
+class Logger:
+    @staticmethod
+    def set_logger():
+        logger.remove()
```
Comment (TA-user) [inline utils/logging.py:7]: why remove is needed here?

File: utils/logging.py [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+        return logger.add("debug.log", format="{time:DD:MM:YYYY:h:m:s} {level} {message}", level="DEBUG",
```
Comment (TA-user) [inline utils/logging.py:8]: would be nice to have different files with different levels debug, error for example

**Review by relaxing-observer — APPROVED** [review]

File: utils/browser_helper.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+class ElementInteractions:
+    Logger.set_logger()
+
+    def __init__(self, browser):
+        self.browser = browser
+
+    def find_element(self, selector):
+        try:
+            element = self.browser.element(selector).should(be.visible)
```
Comment (TA-user) [inline utils/browser_helper.py:15]: generally should is used as an assertion, why  is it used in browser_helper?

File: utils/browser_helper.py [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+            element.should(be.blank).type(text)
+        except TimeoutException:
+            logger.error(f'Cannot send {text} in element "{selector}"...')
+            raise
+        logger.debug(f'{text} is sent in element "{selector}"...')
+
+    def clear_field(self, selector):
+        element = self.find_element(selector)
+        try:
+            element.should(be.blank).clear()
```
Comment (TA-user) [inline utils/browser_helper.py:43]: should(be.blank).clear() - what if it is not blank?

**Review by TA-user — CHANGES_REQUESTED** [review]

File: utils/logging.py (hunk 4 above)
Comment (yefim-mironyuk) [reply utils/logging.py:8]: There is no need for it. 
You said several times that we need only low level logging.
Debug gonna be in file, Error gonna be last string in file.

File: utils/browser_helper.py (hunk 5 above)
Comment (yefim-mironyuk) [reply utils/browser_helper.py:15]: Wouldn't it be better to first check if an element is clickable before clicking it?

File: utils/browser_helper.py (hunk 6 above)
Comment (yefim-mironyuk) [reply utils/browser_helper.py:43]: We can't clear it then i guess)

File: utils/logging.py (hunk 3 above)
Comment (yefim-mironyuk) [reply utils/logging.py:7]: Don't remember exactly, but it was in our course. I always used this consctruction and you've never said something about it.

Comment (yefim-mironyuk) [reply utils/logging.py:7]: Oh. I just remembered. We use it to hide logger messages from terminal, also we should it deletes default logger.

**Pull request merged**